niworkflows/utils/spaces.py

```diff
@@ -363,8 +364,8 @@ class SpatialReferences:
     def check_space(space):
         """Build a :class:`Space` object."""
         try:
-            if isinstance(space[0], Space):
-                return space[0]
+            if isinstance(space, Space):
```

Comment:
I believe ``check_space`` always receives a list of ``Space`` objects.

Reply:
shouldn't this take 1 space at a time anyways? so why would we want a list in that case?